worker/paidAction.js

```diff
@@ -316,13 +316,11 @@ export async function paidActionForwarded ({ data: { invoiceId, withdrawal, ...a
   }, { models, lnd, boss })
 
   if (transitionedInvoice) {
-    const { bolt11, msatsPaid, msatsFeePaid } = transitionedInvoice.invoiceForward.withdrawl
-    // the amount we paid includes the fee so we need to subtract it to get the amount received
-    const received = Number(msatsPaid) - Number(msatsFeePaid)
+    const { bolt11, msatsPaid } = transitionedInvoice.invoiceForward.withdrawl
```

Comment:
Crap. I think the problem here is that I broke the assumption with p2p stuff that `msatsPaid = msatsPaying + fee = payment.mtokens`. I think we'll want to fix that but I don't recall if we need to change stuff elsewhere.

Reply:
It looks like all the withdrawal code no longer includes the `msatsFeePaid` in the `msatsPaid`. I guess we can update any violations of this *new* convention.

Reply:
Actually looks like this is a false alarm. Must've always been this way and this was wrong somehow.